tests/integration/frameworks/paddle/test_paddle_runner.py

```diff
@@ -5,10 +5,11 @@
 
 import bentoml.paddle
 
-from .test_paddle_impl import test_df, train_paddle_model
+# for review: ???
+from .test_paddle_impl import test_df, train_paddle_model  # noqa: F401
```

Comment:
Why do these model imports exist?

Reply:
i don't like having multiple conftest under smaller directory. It is just a way to write test quickly once I created a model.

Reply:
Oh, these are test fixtures, aren't they.

Reply:
yes

Reply:
Hm, seems like the recommended way to make these lint is to add a `conftest.py`. Is there any particular reason you don't like having more than one conftest?

The alternative seems to be that or what I've done here which seems a bit ugly too.